src/bin/helper.rs

```diff
@@ -87,17 +84,29 @@ async fn server(args: ServerArgs) -> Result<(), Box<dyn Error>> {
     };
     let server_config = ServerConfig {
         port: args.port,
-        https: args.https,
+        disable_https: args.disable_https,
         tls,
     };
```

Comment:
`ServerArgs::config()` might be a function that produces a `ServerConfig`, which would simplify this function a little..